Kmozart/free-mentors
# Merge pull request #43 from Kmozart/ft-user-can-create-mentor-session…

#### What does this PR do?
- Have CRUD function CREATE integrated to database.

#### Description of Task to be completed?
- Have users able to create mentor session - requests with mentors.

#### How should this be manually tested?
- ```npm run db:migrate```  to drop and start tables afresh.``` npm run db:migrate-users```  to instantiate the admin to existence and then follow through with ```npm run dev``` .
- Open up on Postman and go through the routes and then check the create session with the authorization from user tokens.

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- #168372417 Users can create mentor sessions - DB endpoint

#### Screenshots (if appropriate)
- N/A

#### Questions:
- N/A
